indico/modules/events/api.py

```diff
@@ -111,6 +112,7 @@ class CategoryEventHook(HTTPAPIHook):
         'subcontributions': 500,
         'sessions': 100,
     }
+    NO_CACHE = True
```

Comment:
Is this a debugging statement you forgot or are we stopping caching altogether for this hook?